src/main/pg/motor.c

```diff
@@ -76,6 +76,7 @@ void pgResetFn_motorConfig(motorConfig_t *motorConfig)
 
 #ifdef USE_DSHOT_BITBANG
     motorConfig->dev.useDshotBitbang = DSHOT_BITBANG_AUTO;
+    motorConfig->dev.useDshotBitbangedTimer = DSHOT_BITBANGED_TIMER_AUTO;
```

Comment:
And maybe add the ability to set the default via a target-level define for non-unified targets?

Reply:
Gah, I'd hoped to have slain target dependent defines in 4.1. But I guess we still have a handful of non-unified targets so we should still add them.